Source/Core/Core/ConfigLoaders/GameConfigLoader.cpp

```diff
@@ -40,155 +44,70 @@ static std::vector<std::string> GetGameIniFilenames(const std::string& id, u16 r
-    {{"Video_Settings", "MSAA"}, {Config::System::GFX, "Settings", "MSAA"}},
-    {{"Video_Settings", "SSAA"}, {Config::System::GFX, "Settings", "SSAA"}},
-    {{"Video_Settings", "EFBScale"}, {Config::System::GFX, "Settings", "EFBScale"}},
-    {{"Video_Settings", "DisableFog"}, {Config::System::GFX, "Settings", "DisableFog"}},
-
-    {{"Video_Enhancements", "ForceFiltering"},
-     {Config::System::GFX, "Enhancements", "ForceFiltering"}},
-    {{"Video_Enhancements", "MaxAnisotropy"},
-     {Config::System::GFX, "Enhancements", "MaxAnisotropy"}},
-    {{"Video_Enhancements", "PostProcessingShader"},
-     {Config::System::GFX, "Enhancements", "PostProcessingShader"}},
-
-    {{"Video_Stereoscopy", "StereoMode"}, {Config::System::GFX, "Stereoscopy", "StereoMode"}},
-    {{"Video_Stereoscopy", "StereoDepth"}, {Config::System::GFX, "Stereoscopy", "StereoDepth"}},
-    {{"Video_Stereoscopy", "StereoSwapEyes"},
-     {Config::System::GFX, "Stereoscopy", "StereoSwapEyes"}},
-
-    {{"Video_Hacks", "EFBAccessEnable"}, {Config::System::GFX, "Hacks", "EFBAccessEnable"}},
-    {{"Video_Hacks", "BBoxEnable"}, {Config::System::GFX, "Hacks", "BBoxEnable"}},
-    {{"Video_Hacks", "ForceProgressive"}, {Config::System::GFX, "Hacks", "ForceProgressive"}},
-    {{"Video_Hacks", "EFBToTextureEnable"}, {Config::System::GFX, "Hacks", "EFBToTextureEnable"}},
-    {{"Video_Hacks", "EFBScaledCopy"}, {Config::System::GFX, "Hacks", "EFBScaledCopy"}},
-    {{"Video_Hacks", "EFBEmulateFormatChanges"},
-     {Config::System::GFX, "Hacks", "EFBEmulateFormatChanges"}},
-
-    // GameINI specific video settings
-    {{"Video", "ProjectionHack"}, {Config::System::GFX, "Video", "ProjectionHack"}},
-    {{"Video", "PH_SZNear"}, {Config::System::GFX, "Video", "PH_SZNear"}},
-    {{"Video", "PH_SZFar"}, {Config::System::GFX, "Video", "PH_SZFar"}},
-    {{"Video", "PH_ZNear"}, {Config::System::GFX, "Video", "PH_ZNear"}},
-    {{"Video", "PH_ZFar"}, {Config::System::GFX, "Video", "PH_ZFar"}},
-    {{"Video", "PH_ExtraParam"}, {Config::System::GFX, "Video", "PH_ExtraParam"}},
-    {{"Video", "PerfQueriesEnable"}, {Config::System::GFX, "Video", "PerfQueriesEnable"}},
-
-    {{"Video_Stereoscopy", "StereoConvergence"},
-     {Config::System::GFX, "Stereoscopy", "StereoConvergence"}},
-    {{"Video_Stereoscopy", "StereoEFBMonoDepth"},
-     {Config::System::GFX, "Stereoscopy", "StereoEFBMonoDepth"}},
-    {{"Video_Stereoscopy", "StereoDepthPercentage"},
-     {Config::System::GFX, "Stereoscopy", "StereoDepthPercentage"}},
-
-    // UI
-    {{"EmuState", "EmulationStateId"}, {Config::System::UI, "EmuState", "EmulationStateId"}},
-    {{"EmuState", "EmulationIssues"}, {Config::System::UI, "EmuState", "EmulationIssues"}},
-    {{"EmuState", "Title"}, {Config::System::UI, "EmuState", "Title"}},
-};
+static const INIToLocationMap& GetINIToLocationMap()
+{
+  static const INIToLocationMap ini_to_location = {
```

Comment:
The GameConfig loader also supports a 

    [System.Section]
    Key=Value

format. This means that keys for new settings do not need to be added here to this conversion map but instead be in that format instead.

This mapping seemed to be intended for the loading of settings in the old format. Perhaps comment(s) should be added to that effect.